src/middlewared/middlewared/main.py

```diff
@@ -867,6 +873,7 @@ def _console_write(self, text, fill_blank=True, append=False):
         giving user at least some basic feedback is fundamental.
         """
         # False means we are running in a terminal, no console needed
+        self.logger.debug('_console_write %r', text)
```

Comment:
Leftover debug

Reply:
It's intentional, I want to see what middleware writes to `/dev/console` when I debug it via SSH

Reply:
You're not merging with that in, right?

Reply:
Why not? I want to see that in the future too.

Reply:
Because thats highly verbose, over 40 entries in the log just because of startup which doesnt add much value.

Reply:
`TRACE` level ok?

Reply:
Yes, thats reasonable.